open-media-match/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -58,33 +59,32 @@ def bank_create():
 @utils.abort_to_json
 def bank_update(bank_name: str):
     # TODO - rewrite using persistence.get_storage()
+    storage = persistence.get_storage()
     data = request.get_json()
-    bank = database.Bank.query.filter_by(name=bank_name).one_or_none()
+    bank = storage.get_bank(bank_name)
+    rename_from: Optional[str] = None
     if bank is None:
         abort(404, "Bank not found")
 
     try:
         if "name" in data:
+            rename_from = bank.name
             bank.name = data["name"]
         if "enabled" in data:
-            bank.enabled = bool(data["enabled"])
+            bank.matching_enabled_ratio = 1 if bool(data["enabled"]) else 0
```

Comment:
nit: This should be taking a float enabled ratio, but this can be a followup PR for someone else

Reply:
I added it as well